exporter/exporterhelper/batch_sender.go

```diff
@@ -0,0 +1,220 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package exporterhelper // import "go.opentelemetry.io/collector/exporter/exporterhelper"
+
+import (
+	"context"
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"go.uber.org/zap"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/exporter"
+	"go.opentelemetry.io/collector/exporter/exporterbatcher"
+)
+
+// batchSender is a component that accepts places requests into batches before passing them to the downstream senders.
+//
+// batch_processor implements consumer.Traces and consumer.Metrics
+//
```

Comment:
I think this is a leftover from copying from https://github.com/open-telemetry/opentelemetry-collector/blob/fc08135156f6de4d553c29f0f5de2b51addbc9c8/processor/batchprocessor/batch_processor.go#L34-L36, so we can remove it, right?
```suggestion
```